scss/app.scss

```diff
@@ -282,6 +284,26 @@ form {
       list-style: none;
       margin:0;
       padding:0 16px;
+      li {
+        padding: 0;
+        border: none;
+        border-top: 2px solid #bac4c8;
```

Comment:
Is there not a colour variable available for `#bac4c8`?

Reply:
There is now.

Reply:
Are none of these acceptable to use? https://github.com/nhsuk/sexual-health-service-finder/blob/master/scss-live/environment/settings/_colours.scss#L9